bin/generate-sweep-files.py

```diff
@@ -264,156 +275,156 @@ def read_region(brickname, filename, bricksdesc):
 
 SWEEP_DTYPE = np.dtype([
 #   ('BRICK_PRIMARY', '?'),
-    ('RELEASE', '>i4'), 
-    ('BRICKID', '>i4'), 
-    ('BRICKNAME', 'S8'), 
-    ('OBJID', '>i4'), 
-    ('TYPE', 'S4'), 
-    ('RA', '>f8'), 
-    ('DEC', '>f8'), 
-    ('RA_IVAR', '>f4'), 
+    ('RELEASE', '>i2'),
+    ('BRICKID', '>i4'),
+    ('BRICKNAME', 'S8'),
+    ('OBJID', '>i4'),
+    ('TYPE', 'S4'),
+    ('RA', '>f8'),
+    ('DEC', '>f8'),
+    ('RA_IVAR', '>f4'),
     ('DEC_IVAR', '>f4'),
-    ('DCHISQ', '>f4', (5,)), 
+    ('DCHISQ', '>f4', (5,)),
     ('EBV', '>f4'),
-#   ('TYCHO2INBLOB', '?'), 
-#   ('OUT_OF_BOUNDS', '?'), 
-#   ('DECAM_FLUX', '>f4', (6,)), 
-#   ('WISE_FLUX', '>f4', (4,)), 
-    ('FLUX_U', '>f4'), 
-    ('FLUX_G', '>f4'), 
-    ('FLUX_R', '>f4'), 
-    ('FLUX_I', '>f4'), 
-    ('FLUX_Z', '>f4'), 
-    ('FLUX_Y', '>f4'), 
-    ('FLUX_W1', '>f4'), 
-    ('FLUX_W2', '>f4'), 
-    ('FLUX_W3', '>f4'), 
-    ('FLUX_W4', '>f4'), 
-#   ('DECAM_FLUX_IVAR', '>f4', (6,)), 
-#   ('WISE_FLUX_IVAR', '>f4', (4,)), 
-    ('FLUX_IVAR_U', '>f4'), 
-    ('FLUX_IVAR_G', '>f4'), 
-    ('FLUX_IVAR_R', '>f4'), 
-    ('FLUX_IVAR_I', '>f4'), 
-    ('FLUX_IVAR_Z', '>f4'), 
-    ('FLUX_IVAR_Y', '>f4'), 
-    ('FLUX_IVAR_W1', '>f4'), 
-    ('FLUX_IVAR_W2', '>f4'), 
-    ('FLUX_IVAR_W3', '>f4'), 
-    ('FLUX_IVAR_W4', '>f4'), 
-#   ('DECAM_MW_TRANSMISSION', '>f4', (6,)), 
-#   ('WISE_MW_TRANSMISSION', '>f4', (4,)), 
-    ('MW_TRANSMISSION_U', '>f4'), 
-    ('MW_TRANSMISSION_G', '>f4'), 
-    ('MW_TRANSMISSION_R', '>f4'), 
-    ('MW_TRANSMISSION_I', '>f4'), 
-    ('MW_TRANSMISSION_Z', '>f4'), 
-    ('MW_TRANSMISSION_Y', '>f4'), 
-    ('MW_TRANSMISSION_W1', '>f4'), 
-    ('MW_TRANSMISSION_W2', '>f4'), 
-    ('MW_TRANSMISSION_W3', '>f4'), 
-    ('MW_TRANSMISSION_W4', '>f4'), 
+#   ('TYCHO2INBLOB', '?'),
+#   ('OUT_OF_BOUNDS', '?'),
+#   ('DECAM_FLUX', '>f4', (6,)),
+#   ('WISE_FLUX', '>f4', (4,)),
+    ('FLUX_U', '>f4'),
+    ('FLUX_G', '>f4'),
+    ('FLUX_R', '>f4'),
+    ('FLUX_I', '>f4'),
+    ('FLUX_Z', '>f4'),
+    ('FLUX_Y', '>f4'),
+    ('FLUX_W1', '>f4'),
+    ('FLUX_W2', '>f4'),
+    ('FLUX_W3', '>f4'),
+    ('FLUX_W4', '>f4'),
+#   ('DECAM_FLUX_IVAR', '>f4', (6,)),
+#   ('WISE_FLUX_IVAR', '>f4', (4,)),
+    ('FLUX_IVAR_U', '>f4'),
+    ('FLUX_IVAR_G', '>f4'),
+    ('FLUX_IVAR_R', '>f4'),
+    ('FLUX_IVAR_I', '>f4'),
+    ('FLUX_IVAR_Z', '>f4'),
+    ('FLUX_IVAR_Y', '>f4'),
+    ('FLUX_IVAR_W1', '>f8'),
+    ('FLUX_IVAR_W2', '>f8'),
+    ('FLUX_IVAR_W3', '>f8'),
+    ('FLUX_IVAR_W4', '>f8'),
+#   ('DECAM_MW_TRANSMISSION', '>f4', (6,)),
```

Comment:
I doubt it was intentional to store the WISE inverse variance fluxes with double precision.  I'll open a new issue.